plugins/reach/lib/kReachManager.php

```diff
@@ -482,7 +482,8 @@ protected function checkPendingEntryTasks($object)
 			$pendingEntryReadyTask->setStatus($newStatus);
 			$pendingEntryReadyTask->setAccessKey(kReachUtils::generateReachVendorKs($pendingEntryReadyTask->getEntryId(), $pendingEntryReadyTask->getIsOutputModerated(), $pendingEntryReadyTask->getAccessKeyExpiry()));
 			if($pendingEntryReadyTask->getPrice() == 0)
-				$pendingEntryReadyTask->setPrice(kReachUtils::calculateTaskPrice($object, $pendingEntryReadyTask->getCatalogItem()));
+				$taskDuration =  $pendingEntryReadyTask->getTaskJobData()->getEntryDuration();
```

Comment:
1. need to wrap the lines with {}
2. remove extra space after =
3. what if getTaskJobData() is null?